### DISCUSSION TITLE
Reset NFC card remotely

### DISCUSSION THREAD

**Discussion by mohaomer12** [nfcgate/nfcgate#134] [Q&A]
Hello,

I am inquiring about the feasibility of remotely resetting a smart card while it's in relay mode. Specifically, I am looking to simulate the action of moving the card away from the phone and then tapping it again. This capability would greatly enhance our testing process, especially in scenarios where the card might be at a distance from the tester.

Your assistance or guidance on this matter would be greatly appreciated.

Thank you for your time and consideration.

**Comment by roussosalex** [comment]
Could you please provide some more details on what you mean by "resetting" a smart card and how that could work "remotely"?

**Reply by mohaomer12** [reply]
In mean when I take NFCGate in the tag mode away from the reader and tap it again to the reader, I need same thing to be emulated in the other side, I mean emulating moving NFCGate in reader mode from the real card and tap it again.
